src/datascience-ui/plot/mainPanel.tsx

```diff
@@ -21,6 +21,9 @@ import { SvgViewer } from '../react-common/svgViewer';
 import { TestSvg } from './testSvg';
 import { Toolbar } from './toolbar';
 
+import { initializeIcons } from '@fluentui/react';
```

Comment:
Comment might be good here (I know this is just a move). Not really obvious to me what this is doing.